_posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.3.0"
+authors:
+- kkondaka
+- dlv
+date: 2023-05-25 21:30:00 -0700
+categories:
+  - releases
+meta_keywords: Data Prepper 2.3.0
+meta_description: Data Prepper 2.3.0 with enhancements to expressions, event tagging, enhancements to add_entries processor, s3 sink and tail sampling processor
+---
+
+Data Prepper 2.3.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release introduces a number of changes that help with Data Prepper’s ability to process complex expressions with functions, arithmetic and string operations.
+
+## Enhancements to Data Prepper expressions
+
+DataPrepper 2.3 supports using functions in expressions. List of functions supported are can be found at https://github.com/opensearch-project/data-prepper/blob/main/docs/expression_syntax.md
```

Comment:
It may be better to put a link in some text.

Also, can we link to the documentation site instead?

https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/

Perhaps:

```
List of supported functions are available in the [Expression syntax documentation](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/).